### PR TITLE
feat(issue-114): updated regex search from match

### PR DESCRIPTION
**Pull request by arkentos**
Issue reference: https://github.com/chaostoolkit/chaostoolkit-kubernetes/issues/114

### CODE REVIEW

File: tests/test_pod.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 def test_terminate_pods_by_name_pattern(cl, client, has_conf):
     has_conf.return_value = False
     pod = MagicMock()
-    pod.metadata.name = "my-app-1"
+    pod.metadata.name = "some-random-string-my-app-1"
```
Comment (sumitnagal) [inline tests/test_pod.py:20]: it is better if you can add these 3 tests, and maintain the existing one for backward compatibility.

Comment (sumitnagal) [reply tests/test_pod.py:20]: please add more tests, and maintain the existing ones.

Comment (arkentos) [reply tests/test_pod.py]: thanks, updated!

**Comment by sumitnagal** [comment]
looks good to me @Lawouach , can you approve it.

**Review by Lawouach — CHANGES_REQUESTED** [review]
LGTM

I know it may be a bit late but could:

* add a changelog entry for the change
* squash your commits into a single one please
Then I'll merge and release :)

**Comment by sumitnagal** [comment]
Looks good to me.

**Comment by arkentos** [comment]
squashed all the commit into one

**Comment by arkentos** [comment]
@Lawouach can you please approve this

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,17 @@
 
 ## [Unreleased][]
 
-[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.25.0...HEAD
+[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.25.1...HEAD
+
+## [0.25.1][] - 2021-02-05
```
Comment (Lawouach) [inline CHANGELOG.md:7]: FYI, we don't set versions in PRs. I'll release as is so that'(s okay but in the future, don't set one please.

**Review by Lawouach — APPROVED** [review]
LGTM

**Pull request merged**